From 1cddb402edde8bc1eecd462907b83fd149aff541 Mon Sep 17 00:00:00 2001
From: "Kumar, Kiran S" <kiran.s.kumar@intel.com>
Date: Tue, 15 Mar 2016 11:52:20 +0530
Subject: [PATCH] hpd_trigger() is not required during pi3 probe.

if hpd_trigger is called during pi3 probe, the state is set
to inversion.

Change-Id: I690fa8cc325ae914afd32201a88087b4f67951d8
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-26660
Signed-off-by: Kumar, Kiran S <kiran.s.kumar@intel.com>
Reviewed-on: https://android.intel.com:443/486910
---
 drivers/misc/pi3usb30532_mux.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/misc/pi3usb30532_mux.c b/drivers/misc/pi3usb30532_mux.c
index 3ff4396..6dc01c4 100644
--- a/drivers/misc/pi3usb30532_mux.c
+++ b/drivers/misc/pi3usb30532_mux.c
@@ -274,7 +274,6 @@ static int pi3usb30532_probe(struct i2c_client *client,
 	/* Get typec edev */
 	chip->edev = extcon_get_extcon_dev("usb-typec");
 	chip->dp_cbl_state = 0;
-	hpd_trigger(chip->dp_cbl_state);
 	schedule_work(&chip->mux_work);
 
 	return 0;
-- 
1.9.1

